tests/mock/api_version.rs

```diff
@@ -3,7 +3,7 @@ extern crate mockito;
 extern crate tokio;
 
 use self::mockito::mock;
-use self::tokio::runtime::current_thread::Runtime;
```

Comment:
Out of curiosity, is there an equivalent in tokio 0.2 to restricting the runtime to the current thread?

Reply:
Are you looking for [`block_on`](https://docs.rs/futures/0.3.1/futures/executor/fn.block_on.html)?

Reply:
Yes, that'll do it!